### PR TITLE
Changelog: added missing entries for deposits pagination and sorting

### PR DESCRIPTION
**Pull request by kalessil**
Adds missing changelog entries for https://github.com/Automattic/woocommerce-payments/pull/1230

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Add - Payment Request Button support for US merchants (Apple Pay, Google Pay, Microsoft Pay, and the browser standard Payment Request API).
 * Update - Not passing level3 data for non-US merchants.
 * Add - REST endpoint for fetching account data.
+* Add - Deposits pagination and sorting
```
Comment (vbelolapotkov) [inline changelog.txt:8]:
```suggestion
* Add - Deposits list pagination and sorting.
```

**Review by vbelolapotkov — CHANGES_REQUESTED** [review]
See suggested phrasing.

File: changelog.txt (hunk 1 above)
Comment (kalessil) [reply changelog.txt]: Done in f93307f22ca0946f84534150ff64e50e391bbd1e

**Comment by kalessil** [comment]
Updated!

**Review by vbelolapotkov — APPROVED** [review]
LGTM!

**Pull request merged**